gaphor/services/diagramexportmanager.py

```diff
@@ -191,7 +189,9 @@ def save_pdf(self, filename, canvas):
     def save_svg_action(self):
         title = "Export diagram to SVG"
         ext = ".svg"
-        diagram = self.main_window.get_current_diagram()
+        diagram = self.component_registry.get_utility(
+            IUIComponent, "diagrams"
+        ).get_current_diagram()
```

Comment:
Should this bit be factored into a separate function/method?

Reply:
Great feedback, I was repeating myself a lot and getting the current diagram is very verbose to look at.